### PR TITLE
A bunch more features.

### PR DESCRIPTION
**Pull request by brett19**

### CODE REVIEW

File: mock/mockimpl/svcimpls/kvimplhello.go [hunk 1]
```diff
@@ -15,18 +15,53 @@ func (x *kvImplHello) Register(h *hookHelper) {
...
+		memd.FeatureJSON,
+		memd.FeatureDuplex,
+		//memd.FeatureClusterMapNotif,
+		memd.FeatureUnorderedExec,
+		//memd.FeatureDurations,
+		//memd.FeatureAltRequests,
+		//memd.FeatureSyncReplication,
+		memd.FeatureCollections,
+		//memd.FeatureOpenTracing,
+		//memd.FeatureCreateAsDeleted,
```
Comment (chvck) [inline mock/mockimpl/svcimpls/kvimplhello.go:47]: I think we do actually support this?

**Comment by chvck** [comment]
Still looks good to me, be nice to actually advertise that we support things :)

File: mock/mockimpl/svcimpls/kvimplhello.go (hunk 1 above)
Comment (brett19) [reply mock/mockimpl/svcimpls/kvimplhello.go:47]: We only halfway support it.  We pick up the flag and pass it through the subdoc executor, but we don't actually use it to force the created document to be deleted.

**Comment by brett19** [comment]
Just waiting on the ability to test this before I push it.

**Pull request merged**